nsxt/utils_test.go

```diff
@@ -27,7 +27,8 @@ import (
 // Default names or prefixed of NSX backend existing objects used in the acceptance tests.
 // Those defaults can be overridden using environment parameters
 const tier0RouterDefaultName string = "PLR-1 LogicalRouterTier0"
-const edgeClusterDefaultName string = "edgecluster1"
+const edgeClusterDefaultName string = "EDGECLUSTER1"
+const computeCollectionDefaultName string = "Cluster-1"
```

Comment:
For local manager environment, this should be Compute-Cluster if I recall (it's the name of the VC cluster which we deploy).

Reply:
BTW the value is set within the Jenkins node but it's better to have a correct default value :)

Reply:
Not sure how that env is deployed. In the test topology `Policy-L2L3-OnPrem` it's called `Cluster-1`.